com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -320,7 +320,7 @@ public bool IsOverlayOpenWithExclusiveInput()
 
             //-------------------------------------------------------------------------
             [Conditional("ENABLE_DEBUG_EOSMANAGER")]
-            static void print(string toPrint)
+            internal static void print(string toPrint, LogType type = LogType.Log)
             {
                 Debug.Log(toPrint);
```

Comment:
I think we should also use the LogType that is passed in 🛩️ 😄 .

Reply:
Implemented as suggested; oops on forgetting in the first place!